src/roguelike/game_engine/game_manager/game_processor/game_state.py

```diff
@@ -0,0 +1,49 @@
+"""
+Describes game state.
+It stores all information about game: current view mode, environment, player and his inventory.
+"""
+
+from enum import Enum, auto
+
+from roguelike.game_engine.env_manager.env_manager import Environment, Inventory
+from roguelike.game_engine.env_manager.map_objects_storage import PlayerCharacter
+
+
+class Key(Enum):
+    W = auto()
+    A = auto()
+    S = auto()
+    D = auto()
+    E = auto()
+    II = auto()
+    M = auto()
+    Q = auto()
+
+
+class Mode(Enum):
+    INVENTORY = auto()
+    MAP = auto()
+
+
+class GameState:
+    """
+    Stores all game parts
+
+    Attributes:
+        mode: current view mode
+        environment: env with objects and map
+        inventory: player's inventory
+        player: main player
+    """
+    def __init__(
+            self,
+            mode: Mode,
+            environment: Environment,
+            inventory: Inventory,
+            player: PlayerCharacter,
+    ) -> None:
+        self.mode = mode
+        self.environment = environment
+        self.inventory = inventory
+        self.player = player
+        self.running = True
```

Comment:
Let's name this one `is_finished` – seems like more obvious naming

Reply:
really? how so? usually such flags are checked in the manner
if running - do smth. is_finished would require additional not
here we would also write something like 
while game.running:
  run_game_turn(state)
i could concede is_running might be better